scipy/optimize/_trustregion_constr/minimize_trustregion_constr.py

```diff
@@ -302,6 +302,10 @@ def _minimize_trustregion_constr(fun, x0, args, grad,
         * 3 : `callback` function requested termination.
         * 4 : Constraint violation exceeds 'gtol'.
 
+        .. versionchanged:: 1.15.0
+            If the constraint violation exceeds gtol then `result.success`
+            will be False. Previously this could've been reported as True.
```

Comment:
```suggestion
            If the constraint violation exceeds `gtol`, then `result.success`
            will now be False.
```

Reply:
Made the commit separately (saw this afterwards)